src/clarity-angular/tree-view/index.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2016 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {Type} from "@angular/core";
+import {TreeView} from "./tree-view";
+import {TreeNode} from "./tree-node";
+
+export * from "./tree-view";
+export * from "./tree-node";
+
+export const TREE_VIEW_DIRECTIVES: Type<any>[] = [
+    TreeNode,
+    TreeView
+];
```

Comment:
You're not exporting the TreeSelection interface! The whole point of it is for consumers to import it and type our outputs.